src/Raven.Studio/wwwroot/App/views/database/indexes/indexes.html

```diff
@@ -183,7 +184,7 @@ <h2 class="on-base-background" data-bind="text: entityName, visible: !groupHidde
 </div>
 
 <script type="text/html" id="index-template">
-    <div class="padding padding-sm">
+    <div data-bind="attr: { class: 'padding padding-sm js-index-template ' + ($root.shouldHighlightIndex(name) ? 'blink-style-basic' : ''), id: 'id_' + name }">
```

Comment:
1. can't we avoid having: id_ in id? It isn't meaningful.
2. What if index name contains characters which are invalid for id?

Reply:
rename ` id_` to `index_`